spec/superset/dashboard/datasets/list_spec.rb

```diff
@@ -117,7 +117,7 @@
     let(:filter_dataset_json) { [] }
     it 'prints a table with the dashboard title and charts' do
       expect(subject.table.to_s).to eq(
-        "+-----+------------------+----------+---------------+------------------+--------+-------------+\n" \
+        "+---------------------------------------------------------------------------------------------+\n" \
```

Comment:
Joining character removed, as this is no longer considered an intersection. See https://github.com/tj/terminal-table/blob/master/History.rdoc#--3